src/util/config/config.cpp

```diff
@@ -120,6 +120,7 @@ namespace dxvk {
     /* Star Wars Battlefront (2015)               */
     { R"(\\starwarsbattlefront(trial)?\.exe$)", {{
       { "dxgi.nvapiHack",                   "False" },
```

Comment:
The change will make the nvapiHack line redundant so might as well also remove that.

Reply:
That makes sense thanks. I'll keep that in mind for the future.

Should be fixed now